packages/plugin-node/src/services/llama.ts

```diff
@@ -11,6 +11,8 @@ import {
     GbnfJsonSchema,
```

Comment:
@jimouris you're probably aware but just in case this shouldn't be included in the PR. Maybe best to stack on https://github.com/elizaOS/eliza/pull/1755?